doc/en/announce/release-3.0.2.rst

```diff
@@ -12,15 +12,13 @@ The changelog is available at http://doc.pytest.org/en/latest/changelog.html.
 
 Thanks to all who contributed to this release, among them:
 
-* Adam Chainz
-* Andrew Svetlov
+* Ahn Ki-Wook
```

Comment:
I only realized that I had picked up the wrong names after I have pushed the package, but fortunately before sending the release announcement to the mailing lists. Oh well. :sweat_smile: